package.json

```diff
@@ -35,10 +35,10 @@
     "develop": "webpack --mode=development --watch & webpack-dev-server --mode=development & wait",
     "build:proto": "./scripts/compile_proto.sh",
     "build:webpack": "webpack --mode='production'",
-    "build:documentation": "typedoc",
+    "build:documentation": "arkit src/ -o ./media/dependencies.png && typedoc --media ./media --gitRevision master --mode modules",
     "build:all": "npm run build:proto && npm run build:webpack && npm run build:documentation",
     "generate:sslcerts": "./scripts/generate_ssl_certs.sh",
```

Comment:
I wonder if this can be part of your test fixtures? If your tests expect there to be TLS certs, then they should be part of the generated certs when starting the test and not. See typescript-demo-lib for global setup and global teardown.

However for development, you could then generate some certs. If we do this, it's best to unify the code that does this between dev and testing.

Reply:
We would want to do property testing too https://github.com/dubzzz/fast-check and xstate-test. @DrFacepalm can provide details on this once he has finished his IntroCept example.

Reply:
I agree, this should go into the test fixtures. But now I'm thinking we have the `CA-Functionality` PR coming up soon which removes this script so perhaps we just take it out?